### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by ddaniels1**

### CODE REVIEW

File: docs/docs/walkthrough/phase-6/Configuring Meal Assist [hunk 1]
```diff
@@ -0,0 +1,18 @@
+Set up Autosensitivity mode prior to meal assist.  This includes pulling dev branch and creating the devices and modifications to reports from the basic setup that will be needed for meal assist. 
+
+Be sure
```
Comment (scottleibrand) [inline docs/docs/walkthrough/phase-6/Configuring Meal Assist]: This looks incomplete.  Did you mean to include this, or was this an aborted effort from before you decided to consolidate them?

File: docs/docs/walkthrough/phase-6/Configure-Automatic-Sensitivity-Mode.md [hunk 2]
```diff
@@ -72,7 +83,11 @@ fields = iob current-temps glucose profile auto-sens meal
 cmd = oref0-determine-basal
 vendor = openaps.vendors.process
 args = 
+```
 
+Note that in the "fields" above that "meal" should only be present if meal assist is configured
+
+```
 [report "enact/suggested.json"]
 profile = [Your Path]/profile.json
```
Comment (scottleibrand) [inline docs/docs/walkthrough/phase-6/Configure-Automatic-Sensitivity-Mode.md]: Might want to change the [Your Path] here...

File: docs/docs/walkthrough/phase-6/Configure-Automatic-Sensitivity-Mode.md [hunk 3]
```diff
@@ -84,6 +99,7 @@ glucose = [Your Path]/glucose.json
 meal = [Your Path]/meal.json
 auto-sens = [Your Path]/auto-sens.json
```
Comment (scottleibrand) [inline docs/docs/walkthrough/phase-6/Configure-Automatic-Sensitivity-Mode.md]: ... and here like you did elsewhere.

File: docs/docs/walkthrough/phase-6/Configure-Automatic-Sensitivity-Mode.md [hunk 4]
```diff
@@ -36,34 +36,45 @@ args = -c "curl -s https://[Your URL]/api/v1/entries.json?count=288 | json -e 't
 
 If your [glucose.json] does not have enough entries you will see a warning when running your auto-sens.json report "Error: not enough glucose data to calculate autosens."
 
-3)	After applying the above change you need to add a new device and report as follow:
+3)	After applying the above change you need to add a new device and report.
+A process device must be added, call it "auto-sense"
```
Comment (scottleibrand) [inline docs/docs/walkthrough/phase-6/Configure-Automatic-Sensitivity-Mode.md]: No "e" on auto-sens.

**Comment by scottleibrand** [comment]
Added some line notes.  There are some other improvements we could make, such as giving people actual commands to run instead of making them edit .ini files, but this is a whole heck of a lot better than nothing, and definitely worth merging ASAP after fixing the few nits I noted.  Thanks!

File: docs/docs/walkthrough/phase-6/Configuring Meal Assist (hunk 1 above)
Comment (ddaniels1) [inline docs/docs/walkthrough/phase-6/Configuring Meal Assist]: Yes abort it 

Sent from my iphone
Excuse the typos

**Comment by ddaniels1** [comment]
Ok will work on it tonight 

Sent from my iphone
Excuse the typos

**Comment by scottleibrand** [comment]
If you'd prefer I can merge and then fix those things in dev.

-Scott

**Comment by ddaniels1** [comment]
Let me try to do it, gives me a better understanding of things to have to explain it. Going to try to start setting up Edison and ti stick tonight. I have all the gear. Any advice beyond what's in the wiki?

Sent from my iphone
Excuse the typos

**Comment by scottleibrand** [comment]
Not really, except not to be afraid to ask questions in Gitter the moment you get stuck.

**Comment by ddaniels1** [comment]
K thanks. I heard the range is better than the carelink.  Really want to be able to keep her rig with us in a fanny and allow her to loop during the day. She is pretty small even to carry the pump. 

Sent from my iphone
Excuse the typos

File: docs/docs/walkthrough/phase-6/Configure-Automatic-Sensitivity-Mode.md (hunk 3 above)
Comment (amazaheri) [inline docs/docs/walkthrough/phase-6/Configure-Automatic-Sensitivity-Mode.md]: Great job @ddaniels1 with updates, one thing I would add is to make sure people double check their full path comparing to guide, reason I originally put [Your Path] :)

**Comment by danamlewis** [comment]
(Waiting to review for the updates, but overall this is looking good, so as soon as recommended updates are made we can merge!)

**Comment by scottleibrand** [comment]
Any objections to merging this now so someone else can fix the remaining things?

**Comment by danamlewis** [comment]
That's fine as long as you make a note that it's known to need further
fixing after it is in.

On Wednesday, May 18, 2016, Scott Leibrand <notifications@github.com> wrote:

> Any objections to merging this now so someone else can fix the remaining
> things?
>
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub
> <https://github.com/openaps/docs/pull/166#issuecomment-220218660>
>


-- 
Dana Lewis | http://www.twitter.com/danamlewis |
http://www.linkedin.com/in/danalewis
"Doing something for someone else is more important than anything you would
do for yourself."

**Comment by ddaniels1** [comment]
No problem to go ahead and merge, I will try to get back to it soon myself,
just caught with trying to set up the TI now

On Wed, May 18, 2016 at 8:25 PM, Dana Lewis <notifications@github.com>
wrote:

> That's fine as long as you make a note that it's known to need further
> fixing after it is in.
>
> On Wednesday, May 18, 2016, Scott Leibrand <notifications@github.com>
> wrote:
>
> > Any objections to merging this now so someone else can fix the remaining
> > things?
> >
> > —
> > You are receiving this because you commented.
> > Reply to this email directly or view it on GitHub
> > <https://github.com/openaps/docs/pull/166#issuecomment-220218660>
> >
>
>
> --
> Dana Lewis | http://www.twitter.com/danamlewis |
> http://www.linkedin.com/in/danalewis
> "Doing something for someone else is more important than anything you would
> do for yourself."
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/openaps/docs/pull/166#issuecomment-220218751>
>



-- 
David V. Daniels, MD
PAMF Interventional Cardiology

C - 415-385-9243

**Pull request merged**